Makefile

```diff
@@ -1024,3 +1028,31 @@ ansible_batcher_deploy: ## Deploy the Batcher. Parameters: INVENTORY, KEYSTORE
 	@ansible-playbook infra/ansible/playbooks/batcher.yaml \
 		-i $(INVENTORY) \
 		-e "keystore_path=$(KEYSTORE)"
+
+ansible_aggregator_create_env: ## Create empty variables files for the Aggregator deploy
+	@cp -n infra/ansible/playbooks/ini/config-aggregator.ini.example infra/ansible/playbooks/ini/config-aggregator.ini
+	@echo "Config files for the Aggregator created in infra/ansible/playbooks/ini"
+	@echo "Please complete the values and run make ansible_aggregator_deploy"
+
+ansible_aggregator_deploy: ## Deploy the Operator. Parameters: INVENTORY
+	@if [ -z "$(INVENTORY)" ] || [ -z "$(ECDSA_KEYSTORE)" ] || [ -z "$(BLS_KEYSTORE)" ]; then \
+		echo "Error: INVENTORY, ECDSA_KEYSTORE, BLS_KEYSTORE must be set."; \
+		exit 1; \
+	fi
+	@ansible-playbook infra/ansible/playbooks/aggregator.yaml \
```

Comment:
This is missing some parameters: `@ansible-playbook infra/ansible/playbooks/aggregator.yaml -i $(INVENTORY) -e -e "ecdsa_keystore_path=$(ECDSA_KEYSTORE)" -e "bls_keystore_path=$(BLS_KEYSTORE)"`

Reply:
Good catch! 👀

Reply:
Fixed in f83f955aaa0df2416e00b11af4eaaea211abb505